Makefile

```diff
@@ -13,11 +13,11 @@ all:
 
 
 release_linux: 
-	dep ensure
+    go mod download
 	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags "-s -w" -o "bin/pushprom-$(VERSION).linux-amd64/pushprom" github.com/messagebird/pushprom
```

Comment:
Should we use the same build command here as in the Docker container?

Reply:
Yes, is the same command because in both cases we are generating a binary for linux, but the "release_linux" is no longer part of the "container" in the make file

Reply:
I was thinking about the slight differences between this command and the command in the Dockerfile (e.g. `-installsuffix cgo`). Should those be similar, or it doesn't matter, as the output of both command is the same (despite the differences)?

Reply:
That is not really necessary, copie from somewhere else :s